### DISCUSSION TITLE
Нет лога

### DISCUSSION THREAD

**Discussion by walkman99** [Anonym-tsk/nfqws-keenetic#51] [Q&A]
После установки последней версии не создается лог в /opt/var/log. Если включить в настройках log_level=1, то в системный лог пишется.

**Comment by Anonym-tsk** [comment]
Покажите конфиг

**Reply by walkman99** [reply]
Нашел причину - моя ошибка. Было задвоение настроек в конфиге, одна строчка не была закомментирована.
